schedule/staging/ext4@uefi-staging.yaml

```diff
@@ -3,27 +3,43 @@ name:           ext4@uefi-staging
 description:    >
   Test for ext4 filesystem.
 vars:
-  FILESYSTEM: ext4
```

Comment:
Why removed this setting for ext4? It is now by default or already set in other file?

Reply:
In the YAML schedule we have this line:

`   - installation/partitioning/guided_setup/select_filesystem_option_ext4
`

That sets ext4 explicitely.

Reply:
Got it, thanks.